File: funnel.client/src/app/interfaces/asistenteOperacion/consultaAsistente.ts
export interface ConsultaAsistente {
    exitoso: boolean;
    error_mensaje: string;
    id_bot: number;
    pregunta: string;
    respuesta: string;
    tokens_entrada: number;
    tokens_salida: number;
    id_usuario: number;
  }
  
  export interface ConsultaAsistenteDto {
    exitoso: boolean;
    errorMensaje: string;
    idBot: number;
    pregunta: string;
    fechaPregunta: Date;
    respuesta: string;
    fechaRespuesta: Date;
    tokensEntrada: number;
    tokensSalida: number;
    idUsuario: number;
    idTipoUsuario: number;
    idEmpresa: number;
    esPreguntaFrecuente: boolean;
  }

File: Funnel.Models/Dto/ConsultaAsistente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Funnel.Models.Dto
{
    public class ConsultaAsistente
    {

        public bool Exitoso { get; set; }
        public string ErrorMensaje { get; set; } = string.Empty;
        public int IdBot { get; set; }
        public string Pregunta { get; set; } = string.Empty;
        public DateTime FechaPregunta { get; set; }
        public string Respuesta { get; set; } = string.Empty;
        public DateTime FechaRespuesta { get; set; }
        public int TokensEntrada { get; set; }
        public int TokensSalida { get; set; }
        public int IdUsuario { get; set; }
        public int IdTipoUsuario { get; set; }
        public int IdEmpresa { get; set; }
        public Boolean EsPreguntaFrecuente { get; set; }
    }
}
